<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_kotoba</class>
 <widget class="QDialog" name="modify_kotoba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>見出し語修正</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/imgs/icon.png</normaloff>:/imgs/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="modify_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修正</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>内容 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mp3_path">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="imi_line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>192</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>備考 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>316</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>音声ファイル :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音声添付</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>漢字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kanji_line">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="bikou_line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="kotoba_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="del_file">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>344</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ファイル削除</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
